News/DataAccess/Repo/ArticleRepo.cs

```diff
@@ -5,12 +5,39 @@ namespace News.DataAccess.Repo;
 
 public class ArticleRepo : IArticleRepo
 {
-    private readonly DbSet<ArticleEntity> _dbSet;
     private readonly DbContext _context;
 
     public ArticleRepo(DbContext context)
-    {
-        _dbSet = context.Set<ArticleEntity>();
+    {        
         _context = context;
+    }        
+
+    public ArticleEntity Add(ArticleEntity entity)
+    {
+        var result = _context.Add(entity);
+        _context.SaveChanges();
+
+        return result.Entity;
```

Comment:
We should discuss will we create a var result and then return it or return it like a result of expression.